features.md

```diff
@@ -41,7 +41,6 @@ and style issues.
 clangd respects your project's `.clang-tidy` file which controls the checks to
 run. Not all checks work within clangd.
 
-
```

Comment:
again, these breaks are deliberate as this page has a lot of sections and subsections and they can be hard to visually distinguish. Do you find the opposite?